packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.tsx

```diff
@@ -43,16 +44,15 @@ export function ComboboxContentList(props: ComboboxListProps): JSX.Element {
             );
           })}
 
-          {props.loading && optionsExist && (
-            <div
-              className={classnames([
-                styles.loadingContainer,
-                styles.hasOptions,
-              ])}
-            >
-              <Spinner size="small" />
-            </div>
-          )}
+          <div
+            className={classnames([styles.loadingContainer, styles.hasOptions])}
+          >
+            <AnimatedPresence transition="fromBottom">
+              {Boolean(props.loading && optionsExist) && (
+                <Spinner size="small" />
+              )}
+            </AnimatedPresence>
```

Comment:
also added some pizzazz when it shows and hides. It slides from the bottom

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdml1ZnMxZ2RvZGlrMnFmMWdkb2E3ajBkYmppcXlxb3c1MjY1bDVtdCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xUA7aSyLZI1SwNS1HO/giphy.gif)